pkgs/tools/text/ruby-zoom/default.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv {
+  name = "ruby-zoom-4.1.0";
+
+  inherit ruby;
+  gemfile = ./Gemfile;
+  lockfile = ./Gemfile.lock;
+  gemset = ./gemset.nix;
+
+  meta = with lib; {
+    description = "Quickly open CLI search results in your favorite editor!";
+    homepage    = https://gitlab.com/mjwhitta/zoom;
+    license     = with licenses; gpl3;
+    platforms   = platforms.unix;
```

Comment:
Yes. I have added myself as the maintainer for this package.

I have already tried adding ncurses as the build input but it gives me the same failure.  Please see the second commit below. I hope I am doing it correctly. 

Do I need to modify gem sources to point to the installed location of ncurses?